### DISCUSSION TITLE
数据库可以支持sqlite吗？

### DISCUSSION THREAD

**Discussion by zensea** [Safe3/uusec-waf#111] [Q&A]
当前可以支持sqlite吗？
如果不能，后续会支持sqlite吗？

**Comment by Safe3** [comment]
暂不支持，以后可能会

**Reply by zensea** [reply]
使用sqlite就不用依赖mysql，更轻量化，再支持stream四层代理。简直是单机waf神器，会吸引一大批小站长。用户量*转化率
